processor/src/advice/mem_provider.rs

```diff
@@ -177,6 +143,6 @@ impl MemAdviceProvider {
 
     /// Returns true if the merkle set with the specified root is present in this advice provider.
     pub fn has_merkle_set(&self, root: Word) -> bool {
-        self.sets.contains_key(&root.into_bytes())
+        self.store.get_node(root, NodeIndex::root()).is_ok()
     }
```

Comment:
Do we need this any more? We should either delete it or rename it as we don't have Merkle sets any more.

Reply:
If this is used for tests only, should we add an attribute to it?